docs/package.json

```diff
@@ -3,9 +3,11 @@
   "version": "2024.08.01",
   "private": true,
   "scripts": {
+    "lint": "npx @biomejs/biome check .",
+    "lint:fix": "npx @biomejs/biome check --write .",
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "docusaurus build",
+    "start": "TYPEDOC_WATCH=true docusaurus start",
+    "build": "TYPEDOC_WATCH=false docusaurus build",
```

Comment:
Followed these docs to set up proper TypeScript reference doc re-generation watching: https://typedoc-plugin-markdown.org/plugins/docusaurus/watch-mode